src/pylogctx/django.py

```diff
@@ -25,3 +25,13 @@ def process_response(self, request, response):
     def process_exception(self, request, exception):
         context.clear()
         return None
+
+
+class ExtractRequestContextMiddleware(OuterMiddleware):
+    def process_request(self, request):
+        super(ExtractRequestContextMiddleware, self).process_request(request)
```

Comment:
Are you sure that you want to inherit ExtractRequestContextMiddleware from OuterMiddleware? It seems like these two middlewares are actually two different API's that do the same thing.
So if I want to use request_extractor then I most likely will not use log_adapter and vise versa.